scripts/approve-superproposal.ts

```diff
@@ -0,0 +1,56 @@
+import { getPod } from '../src';
+import { adminPodAddress, dummyAccount, subPodAddress } from '../env.json';
+import { setup, sleep } from './utils';
+
+async function main() {
+  const { walletOne, walletTwo } = setup();
+
+  const adminPod = await getPod(adminPodAddress);
```

Comment:
we should be calling this `superPod` right? thats the relationship we're testing